[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]

salilsdesai opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64269
* #64268

Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that @jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

Commit:
[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]

Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that @jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

iseeyuan reviewed (approved):
Nice. I like simple code!

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that @jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]

Commit:
Update on "[PyTorch Edge][Model Loading] Operator Call De-dup at TorchScript Serialization Level [2/2]"


Revert changes in D29826210 (we don't need operator lambda caching since there aren't duplicate operators anymore)

This diff stack results in an additional approx 12% speedup in model loading time (from 229ms to 200ms) when run against an 87MB speech model that jiatong provided.

Differential Revision: [D30615710](https://our.internmc.facebook.com/intern/diff/D30615710/)

[ghstack-poisoned]
